test/index.spec.js

```diff
@@ -4,24 +4,28 @@
 const chai = require('chai')
 const { expect } = chai
 chai.use(require('dirty-chai'))
-const IPFSFactory = require('ipfsd-ctl')
+const Ctl = require('ipfsd-ctl')
 const PeerID = require('peer-id')
 
 const DelegatedPeerRouting = require('../src')
-const factory = IPFSFactory.create({ type: 'go' })
+const factory = Ctl.createFactory({ type: 'go', test: true})
 
 async function spawnNode (boostrap = []) {
-  const node = await factory.spawn({
-    // Lock down the nodes so testing can be deterministic
-    config: {
-      Bootstrap: boostrap,
-      Discovery: {
-        MDNS: {
-          Enabled: false
-        }
-      }
-    }
-  })
+  // const node = await factory.spawn({
+  //   // Lock down the nodes so testing can be deterministic
+  //   ipfsOptions: {
+  //     config: {
+  //       Bootstrap: boostrap,
+  //       Discovery: {
+  //         MDNS: {
+  //           Enabled: false
+  //         }
+  //       }
+  //     }
+  //   }
+  // })
+  const node = await factory.spawn()
```

Comment:
@hugomrdias can I have your 👀here? 🙏 

Somehow, I get `SyntaxError: Unexpected end of JSON input` when trying to spawn the daemon. I changed this according to the `ctl` readme instructions, but I cannot get it working